README.md

```diff
@@ -113,6 +113,12 @@ pywis-pubsub publish --topic origin/a/wis2/centre-id/data/core/weather --config
 # publish a message with a WCMP2 metadata id
 pywis-pubsub publish --topic origin/a/wis2/centre-id/data/core/weather --config pub-local.yml -i stationXYZ-20221111085500 -u https://example.org/stationXYZ-20221111085500.bufr4 -g 33.8,-11.8,8.112 -w 0-20000-12345 --metadata-id "x-urn:wmo:md:test-foo:htebmal2001"
```

Comment:
the original command without specifying --datetime will result in a message with properties['datetime'] = None , is that correct ?

Reply:
Yes, per Requirement 9A.